### ISSUE TITLE
Game crashes when starting a new game

### ISSUE THREAD

**Issue by AnunnakiNibiru** [tsunamayo/Starship-EVO#4158]
Hi.

I have probably the same problem as user "aRasid-cz" here: https://github.com/tsunamayo/Starship-EVO/issues/4133 .

I 
[Starship EVO crash related files.zip](https://github.com/tsunamayo/Starship-EVO/files/7116116/Starship.EVO.crash.related.files.zip)
bought the game today, and I am unable to get pass character creation window ("Steam" >>> "Starship Evo" >>> (game starts fine) >>> "Play" >>> "Create a new character"). The "Character Selection" window is empty, so I am forced to create new one.
I put in name (every time I need to add another one, first one was "Anunnaki Nibiru", because previous one is occupied, and there is seed shown.
C:\Users\rbilk\OneDrive\Desktop\Starship EVO\AppData.LocalLow.Backup\Save_Data\Characters\Anunnaki Nibiru
"Instances"
"PlayerEntities"
Both are empty.
When I click on the button "Play with this Character", it seems the game is doing something (shows cartoon image), but after maybe 5 seconds, new small window named "Starship EVO - Unity xxxx (version?)" appears ... on it progressing progressbar, and both windows disappears (crash to desktop).

It happens every time.
I tried 2K and 4K resolution, AA and no AA, windows and fullscreen, no difference, always the same crash to desktop on the same spot.
I tried first original game, than switched to [experimental] (early-access 21W35a). Also tried to delete folder: "C:\Users\rbilk\AppData\LocalLow\Moonfire Entertainment\Starship EVO", even reinstalling game and checking via STEAM for corrupted files.
Still, the same issue every time.

I attached 7zip file with crash/error/logs.

I am experienced in PC/HW/OS/Security/Programming/Networks and etc., so if there is any way how I can help you investigate this issue let me know. Even my English is not as good, I can share my screen with you if you wish to see it or "troubleshoot" it on my PC, anything that can help you make progress with this phenomenal game! (via Zoom, TeamViewer, Webex, or any other you chose).

(latest Windows10, all updates, latest graphic card drivers/sw, no issues in any other program/games).

Starship EVO crash related files.zip

**Comment by tsunamayo** [comment]
@AnunnakiNibiru Hi, I will keep you posted on the dump analysis, thanks for sending it. It could be the same issue indeed, lets hope the next version will fix this. Thanks

**Comment by AnunnakiNibiru** [comment]
Hi, just tested 21W36a, no change :(, exactly the same problem remains.

**Comment by AnunnakiNibiru** [comment]
Hi .... change of state!
After today latest test of 21W36a, which ended the same way as previous one, I later discovered that the game is still running "somewhere" in background.
I restarted computer again. Then I started the latest experimental 21W36a, and in option I selected:
1. Graphics options: 1920x1080@60Hz, Windows, V-Sync Enabled, Head Bobbing 0%, everything else DISABLED.
2. Sound options: Game Volume 100%, Music Volume 32%
3. Game (Player Customization) I selected Name "Anunnaki Nibiru", Race "Human Male" and Hair "Rocker"
4. Entered also in options CONTROLS and KEY MAPPING but did not change anything there.
After that,  "Play" >>> "Create a new character" ... the same as before, .... but this time ... I AM IN THE GAME!

So, I don't know, if you solved the problem in latest experimental version, or maybe some graphic changes which I switched to DISABLED solved this problem.
Previously, when I created this ticket, I did not try to disable all graphic options, other things I tested even before.

So for me right now, it seems it work. I will do few other tests and let you know.
